docs/user/collaboration.rst

```diff
@@ -0,0 +1,81 @@
+.. _collaborating:
+
+Collaborating on Renku
+======================
+
+Beyond building and maintaining your own data science workflows, there are many
+ways to use Renku to share your project with collaborators.
+
+Finding & accessing public projects
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To find and access projects on the Renku platform that you might want to
+interact with, you can click the **Projects** link in the menu bar followed by
+the **Search** tab. Without entering any search terms, you will see a list of
+all public projects. You can click on the project names to be taken to the
```

Comment:
```suggestion
all projects you have access to. You can click on the project names to go to the
```

Reply:
hmm -- I think I was making a distinction between "public"  = "everyone can read", and "access" = "you can run these notebooks". I should check this everywhere and add a third term.

Reply:
the search shows you all the projects you can read - it might mean you have guest access, so you still won't be able to launch notebooks, but you can see the project and presumably fork it.